src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -33,9 +35,6 @@
     @Column(length = 16, nullable = false)
     private Long caseId;
 
-    @Column(length = 70, nullable = false)
-    private String caseTypeId;
-
```

Comment:
Why are we moving caseTypeId from the entity , is it no longer needed ?

Reply:
Was recently removed from the Data Model

https://tools.hmcts.net/confluence/display/RCCD/Case+Payment+Orders#CasePaymentOrders-DataModel

This work is also being done as a separate ticket (https://tools.hmcts.net/jira/browse/CPO-21), so likely that I will wait for that to be merged before merging this PR

Reply:
CPO-21 has been merged to master, and pulled into this ticket in commit https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/aee9fd201481fa9e4808e08cc2a1bb2ba282fa78